app/views/home/index.html.slim

```diff
@@ -1,54 +1,25 @@
 .container.favorite-space
-  .title
-    h2.extra-bold.p--giant Explore In Your Way
-  .row
-    .col-md-3 style="width: 308px;"
-      div
-        div style="width: 100%; display: inline-block;"
-          .triptype
-            = link_to '', "https://www.luxstay.com/vi/s/?property_type=31&amp;property_type=1",
-              class: "triptype__link"
-            .triptype__cover
-              = image_tag "location_6"
-            .triptype__title
-              p.bold.p--large.mb--0 Căn Hộ
-    .col-md-3 style="width: 308px;"
-      div
-        div style="width: 100%; display: inline-block;"
-          .triptype
-            = link_to '', "https://www.luxstay.com/vi/s/?property_type=31&amp;property_type=1", class: "triptype__link"
-            .triptype__cover
-              = image_tag "location_7"
-            .triptype__title
-              p.bold.p--large.mb--0 Biệt Thự
-    .col-md-3 style="width: 308px;"
-      div
-        div style="width: 100%; display: inline-block;"
-          .triptype
-            = link_to '', "https://www.luxstay.com/vi/s/?property_type=31&amp;property_type=1", class: "triptype__link"
-            .triptype__cover
-              = image_tag "location_8"
-            .triptype__title
-              p.bold.p--large.mb--0 Nhà Riêng
-    .col-md-3 style="width: 308px;"
-      div
-        div style="width: 100%; display: inline-block;"
-          .triptype
-            = link_to '', "https://www.luxstay.com/vi/s/?property_type=31&amp;property_type=1", class: "triptype__link"
-            .triptype__cover
-              = image_tag "location_9"
-            .triptype__title
-              p.bold.p--large.mb--0 Studio
-  .title-location
-    h2.extra-bold.p--giant Top Destinations
-  .row style="width: 246px;"
-    div
-      = link_to "#", style: "width: 100%; display: inline-block;"
-        .product.is-relative
-          .product__cover
-            = image_tag "location_2", class: "product__image image-tag"
-          .product__content.is-absolute.white-deep
-            .product__title.extra-bold Hanoi
-            span.product__price.d-block
-              b 2108
-              | Listings
+  .title-favorite-space
+    h2.extra-bold.p--giant = t(".extra-bold")
+    p = t(".description")
+  #exampleSlider
+    .MS-content
+      - @favorite_spaces.each do |favorite_space|
+        .item
+          p
+            = image_tag "location_6"
+            br
+          .article_content
+            .article_content.bold.title-content.text-center
+              = link_to favorite_space.name, favorite_space_path(favorite_space)
+    .MS-controls
+      button.MS-left
+        i.fa.fa-arrow-left
+      button.MS-right
+        i.fa.fa-arrow-right
+
+javascript:
+  $('#exampleSlider').multislider({
```

Comment:
Bỏ vào assets

Reply:
cho vào assets nó sẽ load ở tất cả các trang ở những trang ngoài trang này ra thì sẽ báo lỗi js ạ

Reply:
Cho vào trong asset thì cụ thể bị lỗi gì?

Reply:
`Uncaught TypeError: animateSlideLeft is not a function` nó báo lỗi như vậy và nó báo liên tục khi ở trang khác trang này, em có thử sửa rồi nhưng mà khi sửa lại không chạy được js

Reply:
Cho đoạn JS để vào assets và để trong $(document).ready vẫn bị thế ah

Reply:
cho vào trong `$(document).ready` nó mới bị lỗi ạ, nó mặc định load vì trang nào cũng chạy `layout application`, mà em không tìm cách nào để đoạn js này chỉ chạy cho mỗi `action index` của `home` nên cách này là cách em nghĩ ra ạ

Reply:
đoạn này em không làm, vì em `checkout` qua nên thấy lỗi chỉ nghĩ ra được cách fix như vậy

Reply:
Nếu vậy vẫn cho vào asset nhưng thêm if trước 
```
if ($('#exampleSlider').length > 1) {
...
}
```